src/worker/core/trade/filterItem.ts

```diff
@@ -0,0 +1,19 @@
+export default class FilterItem {
```

Comment:
For consistency, make the file name FilterItem.ts, if it just exports one class FilterItem.